ofl/adventpro/METADATA.pb

```diff
@@ -3,70 +3,41 @@ designer: "Andreas Kalpakidis"
 license: "OFL"
 category: "SANS_SERIF"
 date_added: "2012-02-29"
-fonts {
-  name: "Advent Pro"
-  style: "normal"
-  weight: 100
-  filename: "AdventPro-Thin.ttf"
-  post_script_name: "AdventPro-Thin"
-  full_name: "Advent Pro Thin"
-  copyright: "Copyright (c) 2008 Andreas Kalpakidis (hello@inderesting.com), with Reserved Font Name \"Advent Pro\""
-}
-fonts {
-  name: "Advent Pro"
-  style: "normal"
-  weight: 200
-  filename: "AdventPro-ExtraLight.ttf"
-  post_script_name: "AdventPro-ExtraLight"
-  full_name: "Advent Pro ExtraLight"
-  copyright: "Copyright (c) 2008 Andreas Kalpakidis (hello@inderesting.com), with Reserved Font Name \"Advent Pro\""
-}
-fonts {
-  name: "Advent Pro"
-  style: "normal"
-  weight: 300
-  filename: "AdventPro-Light.ttf"
-  post_script_name: "AdventPro-Light"
-  full_name: "Advent Pro Light"
-  copyright: "Copyright (c) 2008 Andreas Kalpakidis (hello@inderesting.com), with Reserved Font Name \"Advent Pro\""
-}
 fonts {
   name: "Advent Pro"
   style: "normal"
   weight: 400
-  filename: "AdventPro-Regular.ttf"
+  filename: "AdventPro[wdth,wght].ttf"
   post_script_name: "AdventPro-Regular"
-  full_name: "Advent Pro Regular"
-  copyright: "Copyright (c) 2008 Andreas Kalpakidis (hello@inderesting.com), with Reserved Font Name \"Advent Pro\""
+  full_name: "AdventPro Regular"
```

Comment:
@yanone indeed this is not consistent with previous version, full name should be "Advent Pro Regular"